aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -128,12 +128,27 @@ HTTP.sys delegates to kernel mode authentication with the Kerberos authenticatio
 
 ### Configure Windows Server
 
+In the example app and server configuration in this section:
```

Comment:
This sentence and the following two bullets seem unnecessary since each place that uses these addresses explicitly explains them.